pom.xml

```diff
@@ -3430,6 +3433,7 @@
         <hadoop-client-api.artifact>hadoop-client</hadoop-client-api.artifact>
         <hadoop-client-runtime.artifact>hadoop-yarn-api</hadoop-client-runtime.artifact>
         <hadoop-client-minicluster.artifact>hadoop-client</hadoop-client-minicluster.artifact>
+        <scala-maven-plugin.version>4.3.0</scala-maven-plugin.version>
```

Comment:
nit: maybe we should keep the previous warning "Please don't upgrade the version below .."

Reply:
how about "please be very careful" and links to the different JIRAs, and that the 2.7 one must be left as is?

Reply:
+1 for that description, @steveloughran .

Reply:
Yes SGTM too